runtime/doc/news.txt

```diff
@@ -427,6 +427,10 @@ TREESITTER
   code block fence lines vertically.
 • |vim.treesitter.language.inspect()| shows additional information, including
   parser version for ABI 15 parsers.
+• |TSQuery:disable_pattern()| and |TSQuery:disable_capture()| to turn off
+  a specific pattern or capture in a query.
+• |vim.treesitter.get_captures_at_pos()| returns the `pattern_id` of the
+  pattern used to match each capture.
```

Comment:
add to the existing item: https://github.com/neovim/neovim/blob/37786ccfcf07706c7524cdeef321bb4defba45b2/runtime/doc/news.txt#L416